### ISSUE TITLE
support different media track constraints

### ISSUE THREAD

**Issue by luisrudge** [closeio/mic-recorder-to-mp3#15]
`getUserMedia` supports a bunch of [media track constraints](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints). 

Are you interested in this feature? I can submit a PR that adds this as an option.

**Comment by buzinas** [comment]
Sure thing. For backwards compatibilities, we can just default to what we're using today.

**Comment by tluth** [comment]
Hey, was this included in the last release 2.2.2?

**Comment by buzinas** [comment]
Nope, no PRs were opened for that just yet. Accepting PRs though.
